src/app/controllers/inventories.py

```diff
@@ -32,3 +36,35 @@ def list_all_requirements():
         }
 
     return jsonify(return_dados), 200
+
+@inventory.route("/create", methods= ["POST"])
+@requires_access_level("WRITE")
+def create():
+    list_keys = ["product_category_id", "product_code", "title", "value", "brand", "template", "description"]
+    
+    data = allkeys_in(request.get_json(), list_keys)
+    
```

Comment:
Caso o data retorne error, vai dar um problema key error na função exist_product_code(), inserir:

`if 'error' in data:
        return {"error": data}, 401`

Reply:
Definir os argumentos de exist_product_code, assim:

`inv_query = Inventory.query.all()
inventory = intentories_share_schema.dump(inv_query)`

Reply:
são duas linhas ali em cima, está parecendo uma só...